kasa/smartcam/modules/homekit.py

```diff
@@ -0,0 +1,16 @@
+"""Implementation of homekit module."""
+
+from __future__ import annotations
+
+from ..smartcammodule import SmartCamModule
+
+
+class HomeKit(SmartCamModule):
+    """Implementation of homekit module."""
+
+    REQUIRED_COMPONENT = "homekit"
+
+    @property
+    def info(self) -> dict[str, str]:
+        """Not supported, return empty dict."""
```

Comment:
This makes it sound like it's just a placeholder, maybe we should just having this module for now?

Reply:
It’s there for testing whether a device supports HomeKit, ie for the HomeKit plug in so it can exclude devices. If you just mean the `info` property I thought it better to have the attribute that the `Module.HomeKit` type string will return rather than raise a missing attribute error (until we have a common module)

Reply:
Okay 👍